packages/fiori-app-sub-generator/src/fiori-app-generator/fioriAppGeneratorOptions.ts

```diff
@@ -100,4 +100,10 @@ export interface FioriAppGeneratorOptions extends Generator.GeneratorOptions, Fi
      * VSCode Workspace folders
      */
     workspaceFolders?: string[];
+    /**
+     * Used for the post generation task, if set to true, the project will automatically be added to the workspace.
+     *
+     * @default false
+     */
+    addProjectToWorkspace?: boolean;
```

Comment:
Why not have an option `followupCommand`, this would be re-usable in other cases.

Reply:
Enhanced `followupCommand` as discussed